dev-tools/cmd/index_template/index_template.go

```diff
@@ -13,40 +13,41 @@ import (
 // main generates index templates for the beats
 func main() {
 
-	beatName := flag.String("beat.name", "", ": Base index name. Normally {beat_name} (required)")
+	beatVersion := beat.GetDefaultVersion()
+	index := flag.String("index", "", "Base index name. Normally {beat_name} (required)")
 	output := flag.String("output", "", "Required: Full path to the output file (required)")
-	version := flag.String("es.version", beat.GetDefaultVersion(), "Elasticsearch version")
+	version := flag.String("es.version", beatVersion, "Elasticsearch version")
+	file := flag.String("file", "", "Path to fields.yml file")
 
 	flag.Parse()
 
-	var existingFiles []string
-	files := flag.Args()
-
-	if len(files) == 0 {
-		fmt.Fprintf(os.Stderr, "No fields.yml files provided. At least one file must be added.")
+	if len(*file) == 0 {
+		fmt.Fprintf(os.Stderr, "File path cannot be empty")
 		os.Exit(1)
 	}
 
-	if *beatName == "" {
-		fmt.Fprintf(os.Stderr, "beat.name is empty. It must be set.")
+	if *index == "" {
+		fmt.Fprintf(os.Stderr, "index is empty. It must be set.")
 		os.Exit(1)
 	}
 
-	// Skip some of the passed files, as not all beats have the same files
-	for _, f := range files {
-		if _, err := os.Stat(f); err != nil {
-			fmt.Printf("Skipping file because it does not exist: %s", f)
-			continue
-		}
-		existingFiles = append(existingFiles, f)
+	if _, err := os.Stat(*file); err != nil {
+		fmt.Fprintf(os.Stderr, "Input -file does not exist: %s", *file)
```

Comment:
`os.Stat` could fail for reasons other than the file does not exists. You can test for that specific case with [os.IsNotExist](https://golang.org/pkg/os/#IsNotExist) or maybe you could just include the error message from stat in the output.